ui/src/test/java/edu/wpi/grip/ui/codegeneration/tools/JavaPipelineInterfacer.java

```diff
@@ -0,0 +1,124 @@
+package edu.wpi.grip.ui.codegeneration.tools;
+
+import org.opencv.imgcodecs.Imgcodecs;
+
+import java.io.File;
+import java.lang.reflect.InvocationTargetException;
+
+import static org.junit.Assert.fail;
+
+public class JavaPipelineInterfacer implements PipelineInterfacer {
+  private Class pipeline;
+  private Object instance;
+
+  public JavaPipelineInterfacer(String className) {
+    pipeline = PipelineCreator.makeClass(className);
+    try {
+      instance = pipeline.getConstructor().newInstance();
+    } catch (InstantiationException | IllegalAccessException
+        | IllegalArgumentException | InvocationTargetException
+        | NoSuchMethodException | SecurityException e) {
+      e.printStackTrace();
+      fail("Failure to instantiate class " + className);
+    }
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#setSource(int, java.lang.Object)
+   */
+  @Override
+  public void setSource(int num, Object value) {
+    try {
+      pipeline.getMethod("setsource" + num, value.getClass()).invoke(instance, value);
+    } catch (NoSuchMethodException | SecurityException | IllegalAccessException
+        | IllegalArgumentException | InvocationTargetException e) {
+      e.printStackTrace();
+      fail("setsource" + num + " is not valid for class " + value.getClass().getSimpleName() + ""
+          + " because there was a: ");
+    }
+  }
+
+  /* (non-Javadoc)
+   * @see edu.wpi.grip.ui.codegeneration.tools.PipelineInterfacer#setSourceAsObject(int, java
+   * .lang.Object)
+   */
```

Comment:
This is an Overridden method. Javadoc is not needed -- it is automatically added from the super class.